### DISCUSSION TITLE
Creating an empty collection of Eloquent model

### DISCUSSION THREAD

**Discussion by michalkogut** [laravel/framework#32850] [General] [answered]
I've assumed that
`collect(new App\Work()) `
gives me an empty collection of instances of Work model. It does unlit some don't appends atrributes to that model. Then 
`collect(new App\Work())->count() `
returns X, where X is the number of attributes you've added. 

How to get empty collection of instances? My workaround is `App\Work::where('id', 0)->get()` or  `collect(App\Work::find(0))` but is creates unnecessary DB query

**Comment by browner12** [comment]
You need to pass an array to the `collect` method.

```php
collect([new App\Work]);
```

**Comment by michalkogut** [comment]
Nope, it doesn't work:

`>>> collect([new App\Work])->count();`
`=> 1`

**Comment by hailwood** [comment, marked as answer]
`collect` specifically returns an instance of `Illuminate\Support\Collection`.
You want an instance of `Illuminate\Database\Eloquent\Collection`.

So just `$collection = new \Illuminate\Database\Eloquent\Collection()`;
